modules/jpa-basic/src/main/java/io/oasp/module/jpa/dataaccess/api/AdvancedRevisionEntity.java

```diff
@@ -32,7 +33,8 @@
 
   /** @see #getId() */
   @Id
-  @GeneratedValue
+  @GeneratedValue(strategy = GenerationType.AUTO, generator = "native")
+  @GenericGenerator(name = "native", strategy = "native")
```

Comment:
using hibernate proprietary annotations (non JPA) should be avoided. However this is a deeply internal decision of a project that we are taking by code that can not be changed from outside.
IMHO we would have to find a way to configure this via `application.properties` or `orm.xml` instead if the DB is MySQL. After all this is a really messy situation created by MySQL + JPA/Hibernate.